### ISSUE TITLE
Need detailed message when replacing existing order(s)

### ISSUE THREAD

**Issue by lamontfr** [savoirfairelinux/sous-chef#750]
### Expected Behaviour
- When order(s) already exists for a client and date range, and we create new order(s) for the same client and an overlapping date range, a modal dialog should ask the user for confirmation to replace the old order(s) with the new order(s).
- The dialog text should be like : 
  - The following order(s) already exist for **John Doe** : 
    - Order  # 38456, Friday May 19th 2017 :  1x Main dish (Regular), 1x Dessert
    - Order  # 39123, Saturday May 20th 2017 :  1x Main dish (Regular), 1x Green salad
  - Would you like to replace the above orders with : 
    - Order  # 36890, Friday May 19th 2017 : 1x Main dish (Large), 1x Fruit Salad
    - Order  # 36891, Saturday May 20th 2017 : 1x Main dish (Regular), 1x Compote
  - or Cancel ?

### Actual Behaviour
- The dialog text does not give information about the new order(s) and the word "override" is confusing : 

![override_orders_modal](https://cloud.githubusercontent.com/assets/8512608/26280391/43f705b8-3d9f-11e7-8487-faba575a3f68.png)

### Steps to reproduce
1- Click on Kitchen Count ( http://localhost:8000/delivery/order/ )
2- Click on generate orders
3- Note the name of a client that has an order today
4- Click on Orders
5- Click on New order
6- In the select box, choose the client found in step 4
7- In the calendar, click on today's date
8- Click on Place orders and notice the dialog box.

**Comment by JeanFrancoisFournier** [comment]
right... what about when orders are created automatically with the CRON? What happens when order(s) already exists for a client? How does it discriminates between the two conflicting orders when no one is around?
